<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>431</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGraphicsView" name="contenitore_lancette">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>431</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="velocita_text">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>58</y>
        <width>120</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="girimotore_text">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>176</y>
        <width>120</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pedaleacc_text">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>292</y>
        <width>120</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="temperatura_text">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>410</y>
        <width>120</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGraphicsView" name="contenitore_page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="pedaleacc_bar">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>30</width>
        <height>300</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="velocita_text_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>330</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Digital Display</family>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>480</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
